From ba22d57c85b3e94aa8f0fdfb2d03366985e25790 Mon Sep 17 00:00:00 2001
From: Mathew King <mathewk@chromium.org>
Date: Wed, 04 Dec 2019 17:27:47 -0800
Subject: [PATCH] FROMGIT: Input: add privacy screen toggle keycode

Add keycode for toggling electronic privacy screen to the keycodes
definition. Some new laptops have a privacy screen which can be toggled
with a key on the keyboard.

Signed-off-by: Mathew King <mathewk@chromium.org>
Link: https://lore.kernel.org/r/20191017163208.235518-1-mathewk@chromium.org
Signed-off-by: Dmitry Torokhov <dmitry.torokhov@gmail.com>
(cherry picked from commit 25b2f1b77a92b4d850d40eca50d446dd25c09934
git://git.kernel.org/pub/scm/linux/kernel/git/dtor/input.git master)

BUG=b:138397511
TEST=none

Change-Id: Ifb9d75f8b8bff0ac0d8ad91f30f12a47f5b16a86
Reviewed-on: https://chromium-review.googlesource.com/c/chromiumos/third_party/kernel/+/1954586
Reviewed-by: Rajat Jain <rajatja@google.com>
Commit-Queue: Mathew King <mathewk@chromium.org>
Tested-by: Mathew King <mathewk@chromium.org>
Auto-Submit: Mathew King <mathewk@chromium.org>
---

diff --git a/include/uapi/linux/input-event-codes.h b/include/uapi/linux/input-event-codes.h
index 84c024d..0e6e4fa 100644
--- a/include/uapi/linux/input-event-codes.h
+++ b/include/uapi/linux/input-event-codes.h
@@ -649,6 +649,8 @@
  */
 #define KEY_DATA			0x277
 #define KEY_ONSCREEN_KEYBOARD		0x278
+/* Electronic privacy screen control */
+#define KEY_PRIVACY_SCREEN_TOGGLE	0x279
 
 #define BTN_TRIGGER_HAPPY		0x2c0
 #define BTN_TRIGGER_HAPPY1		0x2c0
